release/pipelines/windows-efi-installer/README.md

```diff
@@ -46,16 +46,18 @@ After the ISO is modified it creates a new VirtualMachine which boots from the m
 ```
 
 1. `create-vm-root-disk` creates empty DataVolume which is used for Windows installation.
-2. `import-win-iso` creates new datavolume with Windows ISO file with name defined in `isoDVName` parameter.
+2. `import-win-iso` creates new DataVolume with Windows ISO file with name defined in `isoDVName` parameter. The DataVolume has to be at the same namespace as the PipelineRun (due to the mount of PVC into the PipelineRun pod). 
 3. `modify-windows-iso-file` extracts imported ISO file, replaces prompt bootloader (which is used as a default one when EFI is used) with no-prompt bootloader, pack the updated files back to new ISO, convert the ISO and replaces original ISO with updated one.Replacement of bootloader is needed to be able to automate installation of Windows versions which require EFI.
-4. `create-vm` Task creates a VirtualMachine. A DataVolume with the Windows source ISO will be attached as CD-ROM and a second empty DataVolume will be used as installation destination. A third DataVolume with the virtio-win ISO will also be attached (Pipeline parameter `virtioContainerDiskName`).
+4. `create-vm` Task creates a VirtualMachine. A DataVolume with the Windows source ISO will be attached as CD-ROM and a second empty DataVolume will be used as installation destination. A third DataVolume with the virtio-win ISO will also be attached (Pipeline parameter `virtioContainerDiskName`). The Virtual Machine has to be created in the same namespace as the DataVolume with ISO file. In case you would like to tun VirtualMachine in different namespace, the both Datavolumes have to copied to the same namespace as VirtualMachine.
```

Comment:
```suggestion
4. `create-vm` Task creates a VirtualMachine. A DataVolume with the Windows source ISO will be attached as CD-ROM and a second empty DataVolume will be used as installation destination. A third DataVolume with the virtio-win ISO will also be attached (Pipeline parameter `virtioContainerDiskName`). The VirtualMachine has to be created in the same namespace as the DataVolume with the ISO file. In case you would like to run the VirtualMachine in a different namespace, both Datavolumes have to copied to the same namespace as the VirtualMachine.
```